entity-service/config/detekt/baseline.xml

```diff
@@ -9,25 +9,16 @@
     <ID>LargeClass:EntityServiceTests.kt$EntityServiceTests</ID>
     <ID>LargeClass:Neo4jRepositoryTests.kt$Neo4jRepositoryTests</ID>
     <ID>LongMethod:EntityHandlerTests.kt$EntityHandlerTests$@Test fun `entity attributes update should send two notification if two attributes are updated`()</ID>
-    <ID>LongMethod:EntityService.kt$EntityService$ fun getFullEntityById(entityId: String): JsonLdEntity</ID>
```

Comment:
Are they automatically removed by Detekt or are you doing it manually?

Reply:
with detektBaseline